packages/vendure-plugin-customer-managed-groups/test/customer-managed-groups.spec.ts

```diff
@@ -460,6 +461,80 @@ describe('Customer managed groups', function () {
     expect(updatedGroupMember.emailAddress).toBe('selam.lalem@gmail.com');
   });
 
+  it('Administrators can create new addresses for themselves', async () => {
+    await authorizeAsGroupAdmin();
+    const { myCustomerManagedGroup: group } = await shopClient.query(
+      myCustomerManagedGroupQuery
+    );
+    const groupAdmin = group.customers.find((c: any) => c.isGroupAdministrator);
+    const { updateCustomerManagedGroupMember: newGroup } =
+      await shopClient.query(updateCustomerManagedGroupMemberMutation, {
+        input: {
+          addresses: [
+            {
+              streetLine1: 'Selam Street',
+              countryCode: 'US',
+            },
+          ],
+          customerId: groupAdmin.customerId,
+        },
+      });
+    const authorizedCustomerUpdated = newGroup.customers.find(
+      (c: any) => c.isGroupAdministrator
+    );
+    expect(authorizedCustomerUpdated.addresses).not.toBe(null);
```

Comment:
Its safe to use `toBeDefined`, because `not.toBe(null)` will still allow `undefined`
```suggestion
    expect(authorizedCustomerUpdated.addresses).toBeDefined()
```